cherab/openadas/repository/atomic.py

```diff
@@ -195,3 +236,24 @@ def get_recombination_rate(element, charge, repository_path=None):
     d['rate'] = np.array(d['rate'], np.float64)
 
     return d
+
+
+def get_thermalchargeexchange_rate(element, charge, repository_path=None):
+
+    repository_path = repository_path or DEFAULT_REPOSITORY_PATH
+
+    path = os.path.join(repository_path, 'thermalchargeexchange/{}.json'.format(element.symbol.lower()))
+    try:
+        with open(path, 'r') as f:
+            content = json.load(f)
+        d = content[str(charge)]
+    except (FileNotFoundError, KeyError):
+        raise RuntimeError('Requested recombination rate (element={}, charge={})'
+                           ' is not available.'.format(element.symbol, charge))
```

Comment:
Error message is inaccurate.

Reply:
[changed](https://github.com/Mateasek/cherab_openadas/blob/2ff1cfd8fe2fc8878b398c421c32f93c2a408d8c/cherab/openadas/repository/atomic.py#L258-L260) to:

``` python
    except (FileNotFoundError, KeyError):
        raise RuntimeError('Requested thermal charge-exchange rate (donor={}, donor charge={}, receiver={})'
                           ' is not available.'.format(donor_element.symbol, donor_charge, receiver_element.symbol, receiver_charge))
```